docs/README.md

```diff
@@ -166,6 +185,213 @@ sudo ip link del wg0
+
+### Setup Apex Relay Node
+
+Clone the Apex repository on a VM (or bare metal machine). Apex relay node must be reachable from all the endpoint nodes that want to join the Apex network. Follow the instruction in [Installing the agent](#installing-the-agent) section to setup the node and install the apex binary.
+
+### Create a Relay Enabled Zone
+
+Use the following cli command to create the Relay enabled Apex Zone. You can login to the Apex UI and create the zone as well, ensuring that you toggle the `Hub Zone` switch to on.
+
+```sh
+./apexctl --username=kitteh1 \
+   --password=floofykittens \
+   zone create --name=edge-zone \
+   --hub-zone=true \
+   --cidr=10.195.0.0/24 \
+   --description="Relay enabled zone"
+```
+
+Currently the usernames are hardcoded in the Apex Controller. You can edit `name`, `cidr`, `description` parameters in the CLI commands as per your deployment.
+
+You can list the available zones using following command
+
+```sh
+./apexctl  --username=kitteh1 --password=floofykittens zone list                                                         
+
+ZONE ID                                  NAME          CIDR              DESCRIPTION                RELAY/HUB ENABLED
+dcab6a84-f522-4e9b-a221-8752d505fc18     default       10.200.1.0/20     Default Zone               false
+b130805a-c312-4f4a-8a8e-f57a2c7ab152     edge-zone     10.195.0.0/24     Relay enabled zone         true
+```
+
+You can see the two zones associated with the username `kitteh1`.
+
+### Move user to Relay Enabled Zone
+
+By default an user account is associated with its default zone, that is not a relay enabled zone. So to onboard all the devices to the relay enabled zone, you need to associate that zone as a default zone of the user account.
+
+```sh
+apexctl  --username=kitteh1 --password=floofykittens zone move-user --zone-id b130805a-c312-4f4a-8a8e-f57a2c7ab152
+```
+
+Zone id is printed once you create the zone, or you can get it by listing the zone as mentioned above.
+
+### OnBoard the Relay node to the Relay Enabled Zone
+
+Once the user account is associated with the newly created zone, any OnBoarded device will join the new zone. To OnBoard the Relay node you can use any of the following method
+
+#### Interactive OnBoarding
+
+```sh
+sudo apex --hub-router --stun https://apex.local
+```
+
+It will print an URL on stdout to onboard the relay node 
+
+```
+#sudo apex --hub-router --stun https://apex.local
+Your device must be registered with Apex.
+Your one-time code is: GTLN-RGKP
+Please open the following URL in your browser to sign in:
+https://auth.apex.local/device?user_code=GTLN-RGKP
+```
+
+Open the URL in your browser and provide the username and password that you used to create the zone, and follow the GUI's instructions. Once you are done granting the access to the device in the GUI, the relay node will be OnBoarded to the Relay Zone.
+
+#### Quite OnBoarding
```

Comment:
Quiet?

Reply:
may be `silent onboarding` is better word, open for suggestion here :)

Reply:
I was just pointing out that I think it was a typo - "Quite" vs "Quiet", but I wasn't positive.

"Silent" works too.

... or "Unattended Onboarding" ?

... or "Automated Onboarding" ?

Reply:
:D I didn't notice that. Renamed it to Silent Onboarding.

Reply:
fixed.